registration/api/v2/views.py

```diff
@@ -265,6 +266,45 @@ def theme_config(self, request, pk=None):
 
         return Response(status=status.HTTP_200_OK, data=application.draft_theme_config)
 
+    @action(
+        detail=True,
+        methods=['post'],
+        parser_classes=(MultiPartParser, ),
+        serializer_class=LogoFaviconUploadSerializer
+    )
+    def image(self, request, pk: str):
```

Comment:
@giovannicimolin, can you add a test for this action? Also, though the [DRF documentation](https://www.django-rest-framework.org/api-guide/viewsets/#marking-extra-actions-for-routing) is ambiguous and sounds like the `GET` method is always allowed, 

> The action decorator will route GET requests by default, but may also accept other HTTP methods by setting the methods argument.

from [the code](https://github.com/encode/django-rest-framework/blob/master/rest_framework/decorators.py#L145), it looks like that `GET` is only allowed as a default and the provided methods take precedence over it. Did you test if the `GET` method works for this action? If not, can you test and fix this action appropriately?

CC @alanoe

Reply:
@lgp171188 Making a `GET` request on the endpoint returns:
```
HTTP 405 Method Not Allowed
Allow: POST, OPTIONS
Content-Type: application/json
Vary: Accept

{
    "detail": "Method \"GET\" not allowed."
}
```

Reply:
@giovannicimolin, thanks for checking and confirming. In this case, do we still need the `if` condition since `POST` is the only allowed method?